<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLightSet</class>
 <widget class="QWidget" name="CLightSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>792</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>792</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>792</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,255,0)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,255,0)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(147,147,147)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>140</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border: 0px;background-image: url(:/bin/image/lightSet_up.png);}
QPushButton:pressed{background-image: url(:/bin/image/lightSet_down.png);}
QPushButton:Enabled
{
		color: rgb(255, 255, 255);
}
QPushButton:Disabled
{
	color: rgb(100, 100, 100);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manualBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border: 0px;background-image: url(:/bin/image/lightSet_up.png);}
QPushButton:pressed{background-image: url(:/bin/image/lightSet_down.png);}
QPushButton:Enabled
{
		color: rgb(255, 255, 255);
}
QPushButton:Disabled
{
	color: rgb(100, 100, 100);
}</string>
   </property>
   <property name="text">
    <string>手动</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="subBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>580</y>
     <width>140</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border: 0px;background-image: url(:/bin/image/lightSet_up.png);}
QPushButton:pressed{background-image: url(:/bin/image/lightSet_down.png);}
QPushButton:Enabled
{
		color: rgb(255, 255, 255);
}
QPushButton:Disabled
{
	color: rgb(100, 100, 100);
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(147,147,147)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="autoBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border: 0px;background-image: url(:/bin/image/lightSet_up.png);}
QPushButton:pressed{background-image: url(:/bin/image/lightSet_down.png);}
QPushButton:Enabled
{
		color: rgb(255, 255, 255);
}
QPushButton:Disabled
{
	color: rgb(100, 100, 100);
}</string>
   </property>
   <property name="text">
    <string>自动</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,255,0)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>119</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
